### PR TITLE
Added CHP Bike Siren.

### PR DESCRIPTION
**Pull request by DatedGore**
also changed the horn for both chp sirens

### CODE REVIEW

File: lua/autorun/photon/library/sirens/anemolis_chp_bike.lua [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---Made by: anemolis72
+if not EMVU then return end
+
+EMVU.AddCustomSiren("chp_bike", {
+	Name = "Cal. Highway Patrol Bike Siren",
+	Category = "Other",
+	Set = {
+		{Name = "WAIL", Sound = "emv/sirens/chp bike/emv_wail.wav", Icon="wail"},
+		{Name = "YELP", Sound = "emv/sirens/chp bike/emv_yelp.wav", Icon="yelp"},
+		},
```
Comment (creator2013) [inline lua/autorun/photon/library/sirens/anemolis_chp_bike.lua]: Tab spacing (╯‵□′)╯︵┻━┻

Comment (creator2013) [inline lua/autorun/photon/library/sirens/anemolis_chp_bike.lua]: Tab spacing (╯‵□′)╯︵┻━┻

**Review by creator2013 — CHANGES_REQUESTED** [review]

File: lua/autorun/photon/library/sirens/anemolis_chp_bike.lua (hunk 1 above)
Comment (DatedGore) [reply lua/autorun/photon/library/sirens/anemolis_chp_bike.lua]: 🙄 and which one?

Comment (DatedGore) [reply lua/autorun/photon/library/sirens/anemolis_chp_bike.lua]: oh i see nvm

Comment (DatedGore) [reply lua/autorun/photon/library/sirens/anemolis_chp_bike.lua]: better?

Comment (creator2013) [reply lua/autorun/photon/library/sirens/anemolis_chp_bike.lua]: No (line 10)

**Review by creator2013 — APPROVED** [review]

**Pull request merged**